src/components/DisbursementInviteMessage/index.tsx

```diff
@@ -0,0 +1,156 @@
+import { Fragment, useEffect, useState } from "react";
+import { Card, RadioButton, Textarea } from "@stellar/design-system";
+import { useDispatch } from "react-redux";
+
+import { useRedux } from "hooks/useRedux";
+import { AppDispatch } from "store";
+import { getOrgInfoAction } from "store/ducks/organization";
+
+import "./styles.scss";
+
+interface DisbursementInviteMessageProps {
+  draftMessage?: string;
+  isEditMessage: boolean;
+  onChange?: (message: string) => void;
+}
+
+export const DisbursementInviteMessage = ({
+  draftMessage,
+  isEditMessage,
+  onChange,
+}: DisbursementInviteMessageProps) => {
+  console.log(draftMessage);
+  enum radioValue {
+    ORGANIZATION = "organization",
+    CUSTOM = "custom",
+  }
+
+  const standardOrgMessage =
+    "You have a payment waiting for you from the {{.OrganizationName}}. Click {{.RegistrationLink}} to register.";
+  const customMessagePlaceholder =
+    "Input your registration invitation message for this disbursement here in the appropriate language. Make sure to say which organization the transaction is coming from. Keep your message short. A link to the wallet will be appended at the end.";
+
+  const { organization } = useRedux("organization");
+  const dispatch: AppDispatch = useDispatch();
+
+  const [selectedOption, setSelectedOption] = useState(radioValue.ORGANIZATION);
+  const [customMessageInput, setCustomMessageInput] = useState(
+    customMessagePlaceholder,
+  );
+
+  useEffect(() => {
+    dispatch(getOrgInfoAction());
+  }, [dispatch]);
+
+  const handleOptionChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    if (event.target.value === radioValue.ORGANIZATION) {
+      setCustomMessageInput("");
+      setSelectedOption(radioValue.ORGANIZATION);
+    } else {
+      setSelectedOption(radioValue.CUSTOM);
+    }
+  };
```

Comment:
I don't see where we're updating the parent message (`onChange`) when switching from custom to default message. We probably should clear the custom message when that happens. What do you think?